src/clients/api/queries/getAuthentication/index.ts

```diff
@@ -0,0 +1,17 @@
+import { restService } from 'utilities';
+
+const getAuthentication = async (accountAddress: string | undefined): Promise<boolean> => {
+  if (accountAddress) {
+    const response = await restService({
+      endpoint: `/authentication/${accountAddress}`,
+      method: 'GET',
+    });
+    const authorized = response.status === 200;
+
+    return authorized;
+  }
+
+  return false;
```

Comment:
Also to follow how other query functions are written, I'd return an object:
```suggestion
  if (accountAddress) {
    const response = await restService({
      endpoint: `/authentication/${accountAddress}`,
      method: 'GET',
    });
    const authorized = response.status === 200;

    return {
      authorized,
    };
  }

  return {
    authorized: false,
  };
```